packages/babel-preset-app/index.js

```diff
@@ -40,14 +40,14 @@ module.exports = function() {
 
 			// Polyfills the runtime needed for async/await and generators
 			// This assumes Object.assign is available
```

Comment:
I think a recent update to babel has broken this. Needs to be fixed before merging.

Reply:
Fixed outside of this PR